src/smt/smt_engine.cpp

```diff
@@ -647,7 +631,7 @@ class SmtEnginePrivate : public NodeManagerListener {
 
 }/* namespace CVC4::smt */
 
-SmtEngine::SmtEngine(ExprManager* em)
+SmtEngine::SmtEngine(ExprManager* em, Options* optr)
```

Comment:
Minor question: Why `optr` instead of just `opt`?

Reply:
`optr` is "option pointer" :)